src/tests/modules/ser.rs

```diff
@@ -163,6 +163,7 @@ fn int64() {
 
 #[test]
 fn oid() {
+    let _guard = LOCK.run_concurrently();
```

Comment:
The serde tests are run separately from the `u2i` and regular tests, so this lone concurrent assurance won't do anything on its own. For this reason, I think it's probably fine if we just don't bother acquiring the lock in the serde tests at all.

Reply:
makes sense. fixed

Reply:
So I just noticed that my comment was wrong in some ways. The serde tests in `ser/tests/modules/ser.rs` _do_ need the lock (as was mentioned in my comment below), but the tests in `serde-tests/*` are the ones that don't need it, since they're run separately.

The `u2i` tests also need the locks, since they are run via the same `cargo test` invocation as the rest of the regular tests (see `run-tests-u2i.sh`).